shared/src/commonMain/kotlin/co/nimblehq/blisskmmic/domain/repository/AppInfoRepository.kt

```diff
@@ -0,0 +1,8 @@
+package co.nimblehq.blisskmmic.domain.repository
+
+import kotlinx.coroutines.flow.Flow
+
+interface AppInfoRepository {
+    fun getAppVersion(): Flow<String>
```

Comment:
Same here on the missing line break 
```suggestion
interface AppInfoRepository {

    fun getAppVersion(): Flow<String>
```